mycore-pi/src/main/java/org/mycore/pi/condition/MCRPIPredicateBase.java

```diff
@@ -0,0 +1,33 @@
+package org.mycore.pi.condition;
+
+import java.util.Map;
+
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.common.config.MCRConfigurationException;
+
+public abstract class MCRPIPredicateBase implements MCRPIPredicate {
+
+    public MCRPIPredicateBase(String propertyPrefix){
+        this.propertyPrefix = propertyPrefix;
+    }
+
+    private final String propertyPrefix;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5472577)